viewing-data.html

```diff
@@ -179,17 +179,18 @@
 
 <p>Intensity values can be adjusted using the slider, or entered in the text boxes.</p>
 
-<p>Click on the colour-picker to change the colour or select the colour look up table (LUT) assigned to a channel.</p>
-
-<p>In the colour picker, select the <span class="bold">Reverse Intensity</span> checkbox to reverse the display of colour according to intensity.</p>
-
-
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
 <img alt="../images/viewingDataChannels.jpg" class="align-center" src="images/viewingDataChannels.jpg" style="width: 100%;" />
 
+<p>Click on the colour-picker to change the colour or select the colour look up table (LUT) assigned to a channel.</p>
```

Comment:
Oh, yes, color in the new content here at least would be good, I can Brit-pick the rest of the pages post release for 5.3.1